ZenPacks/zenoss/WBEM/tests/test_twisted_client.py

```diff
@@ -53,13 +53,15 @@ def test_parses(self):
 </INSTANCE>
 </VALUE.NAMEDINSTANCE>
         </xml>''')
-        res = EnumerateInstances.parseResponse(xml)
-        self.assertEqual(len(res), 1)
-        self.assertEqual(res[0].classname, 'Clar_DiskDrive')
-        self.assertEqual(len(res[0].properties), 8)
-
-        self.assertEqual(res[0].properties['OtherInterconnectType'].value, '')
+        try:
+            res = pywbem.twisted_agent.EnumerateInstances.parseResponse(xml)
+            self.assertEqual(len(res), 1)
+            self.assertEqual(res[0].classname, 'Clar_DiskDrive')
+            self.assertEqual(len(res[0].properties), 8)
 
+            self.assertEqual(res[0].properties['OtherInterconnectType'].value, '')
+        except:
+            pass
```

Comment:
What exception are we allowing to disregard all of these assertions?

Reply:
not needed anymore.  I was using that as a pdb.set_trace() when diving into the pickle errors.